qutebrowser/config/parsers/keyconf.py

```diff
@@ -422,7 +422,7 @@ def get_bindings_for(self, section):
 
     def get_reverse_bindings_for(self, section):
         """Get a dict of commands to a list of bindings for the section."""
-        cmd_to_keys = collections.defaultdict(list)
+        cmd_to_keys = {}
```

Comment:
This probably won't work due to the `cmd_to_keys[cmd].append(key)` below, no? I was thinking about still using a `collections.defaultdict`, but then doing `return dict(cmd_to_keys)` to convert it to a normal dict.